customer-config/checkout-app/cha.settings.v1.json

```diff
@@ -184,37 +184,51 @@
     "AutoUpdateType": {
       "description": "This is a definition of the app update maintenance window.",
       "type": "object",
+      "required": [
+        "channel",
+        "maintenanceWindow"
+      ],
       "properties": {
         "channel": {
           "type": "string",
           "enum": [
-            "alpha",
-            "beta",
-            "latest"
+            "internal",
+            "preview",
+            "stable"
           ],
           "description": "The channel for the update."
         },
         "maintenanceExclusion": {
           "type": "object",
-          "description": "The maintenance exclusion window for the update. The exclusion maintenance period takes precedence over any scheduled maintenance window.",
+          "description": "The maintenance exclusion window for the update. The exclusion maintenance period takes precedence over any scheduled maintenance window. Maximum of 30 days allowed. The exlusion window is applied only for stable channel and only for major and minor updates(semantic versioning).",
+          "required": [
+            "start",
+            "duration"
+          ],
           "properties": {
             "start": {
               "type": "string",
               "format": "date-time",
               "example": "2024-01-14T07:00:00.000Z",
               "description": "The start date and time of the maintenance exclusion. Specify in the UTC time zone."
             },
-            "end": {
-              "type": "string",
-              "format": "date-time",
-              "example": "2024-01-15T07:00:00.000Z",
-              "description": "The end date and time of the maintenance exclusion. Specify in the UTC time zone."
+            "duration": {
+              "type": "integer",
+              "minimum": 1,
+              "maximum": 30,
+              "example": "5",
+              "description": "The number of days from the start date for the maintenance exclusion to take precedence."
```

Comment:
what if I set it to 3.4?

Reply:
this is not an integer so you should get an error